docs/api/picker-legacy.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "ion-picker-legacy"
```

Comment:
Threading: Picker Column and Picker Column Option will be autogenerated once v8 is published to the `next` tag, right?

Reply:
The API markdown pages are manually created. The metadata (props, slots, etc.) are auto-generated. I missed those - good catch. I'll get that updated + the other changes.

Reply:
Addressed here: https://github.com/ionic-team/ionic-docs/pull/3321/commits/0c7819ac51c94def8cd756aaa2695e65ec29fca0